tests/validation/converters/test_vision_excel_converter.py

```diff
@@ -297,3 +301,12 @@ def test_log_levels(capsys):
     cvtr5.set_log_level(logging.CRITICAL)
     outerr = capsys.readouterr()
     assert "debug" not in outerr.out
+
+
+def test_uuid_excel_input():
+    source_file = Path(str(SOURCE_FILE_97).format(language="en"))
+    ref_file_97 = convert_guid_vision_excel(source_file, number="Number", terms_changed=terms_chaged)
+    data_native, _ = VisionExcelConverter(source_file, language="en", terms_changed=terms_chaged).load_input_data()
```

Comment:
same here: please do a global search for `"en"` and `"nl"` and replace with `LANGUAGE_EN` and `LANGUAGE_NL` resp

Reply:
I would confine this change only in this PR and not flood the whole code base. We should make another PR for it.

Reply:
ok, please do 😄